waiter/src/waiter/scheduler/kubernetes.clj

```diff
@@ -1292,12 +1343,13 @@
 (defn kubernetes-scheduler
   "Returns a new KubernetesScheduler with the provided configuration. Validates the
    configuration against kubernetes-scheduler-schema and throws if it's not valid."
-  [{:keys [authentication authorizer cluster-name container-running-grace-secs custom-options http-options log-bucket-sync-secs
+  [{:keys [authentication authorizer cluster-name container-running-grace-secs custom-options http-options leader?-fn log-bucket-sync-secs
            log-bucket-url max-patch-retries max-name-length pdb-api-version pdb-spec-builder pod-base-port pod-sigkill-delay-secs
-           pod-suffix-length replicaset-api-version replicaset-spec-builder restart-expiry-threshold reverse-proxy scheduler-name
-           scheduler-state-chan scheduler-syncer-interval-secs service-id->service-description-fn
+           pod-suffix-length replicaset-api-version replicaset-spec-builder restart-expiry-threshold restart-kill-threshold
+           reverse-proxy scheduler-name scheduler-state-chan scheduler-syncer-interval-secs service-id->service-description-fn
            service-id->password-fn start-scheduler-syncer-fn url watch-connect-timeout-ms watch-retries watch-socket-timeout-ms]
     {fileserver-port :port fileserver-scheme :scheme :as fileserver} :fileserver
+    :or {restart-kill-threshold 8}
```

Comment:
How about `(+ 2 restart-expiry-threshold)` instead of `8`? That would avoid breaking existing configurations where restart-expiry-threshold > 8.